main_routers/agent_router.py

```
@@ -0,0 +1,194 @@
+}
+
+INTERNAL_TO_FRONTEND = {
+    'computer_use_enabled': 'enable_cu',
+    'mcp_enabled': 'enable_mcp',
+    'agent_enabled': 'enable_up',
+}
+
+
+@router.post('/flags')
+async def update_agent_flags(request: Request):
+    """来自前端的Agent开关更新，级联到各自的session manager。"""
```

Ruff (0.14.8)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)